### PR TITLE
[BOARD]: CanaryOne

### PR DESCRIPTION
**Pull request by hdngr**
- Adds CanaryOne firmware per [firmware proposal](https://github.com/meshtastic/firmware/issues/3149)
- [Protobuf pr](https://github.com/meshtastic/protobufs/pull/427)

### CODE REVIEW

File: .vscode/extensions.json
Comment (thebentern) [inline .vscode/extensions.json:1]: Looks like this one got accidentally committed.

Comment (hdngr) [reply .vscode/extensions.json:1]: Fixed,

**Review by thebentern — APPROVED** [review]

**Comment by hdngr** [comment]
@thebentern bumped protobuf submodule...  anything else you need on this?  We're picking up  boards next week and should have some to the development team soon.

**Comment by thebentern** [comment]
@hdngr looks good to me. Once you merge or rebase on upstream master, I'll get this one integrated

**Comment by hdngr** [comment]
@thebentern reverted change to []()

My understanding is everything under `src/mesh/generated` is generated from protobuffs in CI build.

-[`update_protobufs`](https://github.com/canaryradio/firmware/blob/5585bf5afe41960afd7325e3a9c9ff1105f669d4/.github/workflows/update_protobufs.yml#L26)
- [`regen_protos`](https://github.com/meshtastic/firmware/blob/master/bin/regen-protos.sh#L11)

cc: @lolsborn

**Review by thebentern — APPROVED** [review]

**Pull request merged**